docs/test-config.md

```diff
@@ -192,6 +195,8 @@ configurations:
       run_time: str # (4)!
       # Test target URL 
       target_host: str # (5)!
+      # * Timeout to end current task (sec) # (11)!
+      termination_timeout: int
```

Comment:
```suggestion
      # Timeout to end current task (sec) 
      termination_timeout: int # (11)!
```
I don't believe you desire the resultant display of that annotation 
<img width="251" alt="image" src="https://github.com/AbdelrhmanHamouda/intensive-brew/assets/7114320/71f67ed5-8265-420e-a635-9778d8150605">
As you see, it removes your comment and leave it looking bad

Reply:
oh, nice catch, thanks